src/github/pullRequestModel.ts

```diff
@@ -120,6 +120,9 @@ export class PullRequestModel extends IssueModel<PullRequest> implements IPullRe
 	private _onDidChangeChangesSinceReview = new vscode.EventEmitter<void>();
 	public onDidChangeChangesSinceReview = this._onDidChangeChangesSinceReview.event;
 
+	public onDidChangeChangesSinceReviewActiveEmitter = new vscode.EventEmitter<void>();
```

Comment:
I'd like to avoid this pattern of having a public emitter. It leaks class internal state (when events should be fired) externally. I've suggested another option in another comment.